content/platforms/kubernetes/stability.md

```diff
@@ -0,0 +1,110 @@
+---
```

Comment:
@alexmilowski Please also see the first few topics here: https://github.com/RedisLabs/redis-enterprise-k8s-docs/blob/master/docs/topics.md

Reply:
That document says the same things so we're not contradicting ourselves. Everything that is after [PSP](https://github.com/RedisLabs/redis-enterprise-k8s-docs/blob/master/docs/topics.md#pod-security-policy-psp) needs to go into a different section. I'll do that via a different PR.